recipes/te-aid/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "TE-Aid" %}
+{% set version = "0.1.0" %}
```

Comment:
Tag is [v.1.0.0](https://github.com/clemgoub/TE-Aid/tree/v.1.0.0)

I guess try setting it as "1.0.0"